src/NzbDrone.Core/Datastore/Migration/004_add_various_qualities_in_profile.cs

```diff
@@ -0,0 +1,263 @@
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using FluentMigrator;
+using Newtonsoft.Json;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(4)]
+    public class add_various_qualites_in_profile : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("UPDATE QualityDefinitions SET Title = 'MP3-160' WHERE Quality = 5"); // Change MP3-512 to MP3-160
+            Execute.WithConnection(ConvertProfile);
+        }
+
+        private void ConvertProfile(IDbConnection conn, IDbTransaction tran)
+        {
+            var updater = new ProfileUpdater3(conn, tran);
+
+            updater.AddQuality(13);
+            updater.MoveQuality(5, 0);
+            updater.CreateNewGroup(0, 1000, "Trash Quality Lossy", new[] { 24, 25, 26, 27, 28, 29, 30, 31, 32 });
```

Comment:
Should these numbers be given as constants somewhere? They seem to be kind of random numbers at first glance.

Reply:
They are quality ids defined in quality.cs

Reply:
I agree with Daniel.. shouldn't we be using ENUMS or Constants to more fluent?

Reply:
Will update